core/src/test/scala/unit/kafka/utils/JsonTest.scala

```diff
@@ -41,14 +42,23 @@ class JsonTest {
     val jnf = JsonNodeFactory.instance
 
     assertEquals(Json.parseFull("{}"), Some(JsonValue(new ObjectNode(jnf))))
+    assertEquals(Json.tryParseFull("{}"), Right(JsonValue(new ObjectNode(jnf))))
+    org.junit.Assert.assertThrows(classOf[IllegalArgumentException], () => Json.tryParseFull(null))
+    org.junit.Assert.assertThrows(classOf[IllegalArgumentException], () => Json.tryParseBytes(null))
```

Comment:
Why can't we just say `assertThrows` here?

Reply:
The `import org.scalatest.Matchers._` also imports an assertThrows here, that's why I used the fully qualified name here, although the scala matcher is only used in line 54 (`tryRes shouldBe a [Left[_, JsonValue]]`), but I can replace it with an instanceOf check and then there will be no need to use the `Matchers`.